validate/validate.go

```diff
@@ -490,6 +499,30 @@ func namespaceValid(ns rspec.Namespace) bool {
 	return true
 }
 
+func nestedValid(os, pathA, pathB string) bool {
+	var purePathA, purePathB string
+	if os == "windows" {
+		purePathA = strings.TrimSuffix(pathA, "\\")
+		purePathB = strings.TrimSuffix(pathB, "\\")
+	} else {
+		purePathA = strings.TrimSuffix(pathA, "/")
+		purePathB = strings.TrimSuffix(pathB, "/")
+	}
+
+	if purePathA == purePathB {
+		return false
+	} else if strings.HasPrefix(purePathB, purePathA) {
+		splitedPaths := strings.SplitN(purePathB, purePathA, 2)
+		if os == "windows" && strings.HasPrefix(splitedPaths[1], "\\") {
```

Comment:
You'll also want to catch the case where `splitedPaths` (`pathComponents`?) has a length of one (because the initial path was `/` or some such).